app/assets/stylesheets/custom.css.scss

```diff
@@ -115,77 +115,125 @@ a:hover, a:active {
 }
 
 #cart:hover {
-    background-color: $ocean-green;
+  background-color: $ocean-green;
 }
 
 #create-button {
-    color: $white;
+  color: $white;
 }
 
 #release {
-    color: $white;
+  color: $white;
 }
 
 .navbar-default {
-    background-color: $parchment;
-    border-color: $parchment;
-    margin: 5px;
-    margin-bottom: 20px;
+  background-color: $parchment;
+  border-color: $parchment;
+  margin: 5px;
+  margin-bottom: 20px;
 }
 
-
+.btn-details {
+  background-color: #ffffff;
+  border-color: $clam-tan;
+  margin-top: 10px;
```

Comment:
Properties should be ordered background-color, border-color, color, margin, margin-top